.github/workflows/php-pr-checks.yml

```diff
@@ -0,0 +1,70 @@
+name: PHPUnit / PHPCS / Phan
+on:
+  pull_request:
+    branches: '**'
+
+  push:
+    branches: [ master, MW_1_37, REL1_39 ]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        php_version: ['8.0']
+        mw: ['REL1_39']
+
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php_version }}
+          # We don't run any coverage, so we should set this parameter to none
+          # as this will disable xdebug which slows all processes significantly
+          coverage: none
+          extensions: ast
+      - uses: actions/checkout@v3
+
+      - name: Checkout Mediawiki
+        uses: actions/checkout@v3
+        with:
+          repository: wikimedia/mediawiki
+          path: mediawiki
+          ref: ${{ matrix.mw }}
+
+      - name: Checkout Echo (dependency) extension
+        uses: actions/checkout@v3
+        with:
+          repository: wikimedia/mediawiki-extensions-Echo
+          path: mediawiki/extensions/Echo
+
+      - name: Checkout Flow (implicit dependency) extension
+        uses: actions/checkout@v3
+        with:
+          repository: wikimedia/mediawiki-extensions-Flow
+          path: mediawiki/extensions/Echo
```

Comment:
this path is not correct :)

Reply:
![image](https://user-images.githubusercontent.com/8492735/229054362-bdfb2936-9eb2-453b-9971-900edd848473.png)